homeassistant/components/blink/camera.py

```diff
@@ -121,6 +123,12 @@ def brand(self) -> str | None:
         """Return the camera brand."""
         return DEFAULT_BRAND
 
+    async def record(self) -> None:
+        """Send record command to camera."""
+        with contextlib.suppress(TimeoutError):
```

Comment:
The best practice is to raise a `HomeAssistantError` when a service call fails. Catch and re-raise?

Reply:
You will need to add translations for the raised error as well...
https://developers.home-assistant.io/docs/internationalization/core/#exceptions

Reply:
I see you were caught by the "do it like the other service" - we need to update the https://github.com/home-assistant/core/blob/7c7ca87a3ef56866e9b83ce20ff4f2b12d4fe34f/homeassistant/components/blink/camera.py#L132-L136 as well - I'll put that on the "list" :)
This is more like how to do it: https://github.com/home-assistant/core/blob/7c7ca87a3ef56866e9b83ce20ff4f2b12d4fe34f/homeassistant/components/blink/camera.py#L106-L111, except add translation to the new one.  I'll also put "general translation" on the list for the rest of the raised errors.

Reply:
Okay, I will look into updating all services as well.

Reply:
I've already made the updates to the other services.  You just need to fix up this one.

Reply:
Thanks, saw that already. Will finalize it tonight, didn't have time to fix CI yet.